### ISSUE TITLE
Switch "More Days" not visible

### ISSUE THREAD

**Issue by primarto** [martykan/forecastie#498]
In Graph switch More Days it is not visible, everything is blue, see :
![fcs](https://user-images.githubusercontent.com/62907157/89096674-ef99ad00-d3c7-11ea-84f9-36d24409ce6c.png)

**Comment by robinpaulson** [comment]
Yep, cheers for this. We can likely fix it by changing one colour in the problematic theme.

By any chance did you check the other themes, to see if there were similar problems?

**Comment by primarto** [comment]
All themes where blue color is present, gray is good.
